apps/dashboard/pages/identity-portal/check-in-info/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { Colors2023 } from '@hibiscus/styles';
+import { BoldText, Modal, Text } from '@hibiscus/ui';
+import { ColorSpan, GlowSpan, Search } from '@hibiscus/ui-kit-2023';
+import searchUser from '../../../common/search-user';
```

Comment:
This file is not going to compile either, I'd recommend excluding it from the PR since it's identity portal stuff and the other identity portal files don't seem to have been included